packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -818,6 +818,30 @@ nodeunitShim({
 
     test.done();
   },
+
+  'cluster arn is correct when cluster identifier has uppercase letters'(test: Test) {
+    const stack = testStack();
+    const clusterIdentifier = 'TestClusterIdentifier';
+    const vpc = ec2.Vpc.fromLookup(stack, 'VPC', { isDefault: true });
+    const cluster = new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_MYSQL,
+      vpc,
+      clusterIdentifier,
+    });
+
+    test.deepEqual(
+      stack.resolve(cluster.clusterArn),
+      {
+        'Fn::Join': [
+          '',
+          [
+            'arn:', { Ref: 'AWS::Partition' },
+            ':rds:us-test-1:12345:cluster:testclusteridentifier',
+          ],
+        ],
+      } );
+    test.done();
+  },
```

Comment:
The formatting here is a little off. I would do something like

```suggestion
        'Fn::Join': ['', [
          'arn:', { Ref: 'AWS::Partition' },
          ':rds:us-test-1:12345:cluster:testclusteridentifier',
        ]],
      });

    test.done();
  },
```